### DISCUSSION TITLE
Setting `busy_timeout` w/ Sqlite3

### DISCUSSION THREAD

**Discussion by davissp14** [aarondl/sqlboiler#1135] [Q&A] [answered]
I'm having some trouble figuring out how to properly set the SQLITE3 `busy_timeout` using this ORM.  Is this currently possible?  If so, what's the goto strategy for handling this?

This is what i've tried, without success: 

```
store, err := sql.Open("sqlite3", "path_to_db_file.db?_busy_timeout=5000")
if err != nil {
 ... 
}

resources, err := state.MyResource().All(ctx, store)
if err != nil {
  ...
}
```

**Comment by stephenafamo** [comment, marked as answer]
Honestly, SQLboiler does not concern itself with your DB connection at all, it just uses the methods provided by the standard library. So your question is better directed to the driver package.
